src/App.tsx

```diff
@@ -1,73 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
```

Comment:
Ensure that `peopleFromServer` is correctly imported and structured as expected. Verify that it is an array of `Person` objects, as required by the `Autocomplete` component.